Library/ViewModels/CommentsViewModel.swift

```diff
@@ -86,8 +86,10 @@ public final class CommentsViewModel: CommentsViewModelType,
       }
 
     self.configureCommentComposerViewWithData = Signal
-      .combineLatest(initialProject.signal, currentUser.signal)
-      .takeWhen(self.viewDidLoadProperty.signal)
+      .combineLatest(initialProject, currentUser.signal, self.viewDidLoadProperty.signal.ignoreValues())
+      .map { project, currentUser, _ in
```

Comment:
Couldn't remove the `_` completly. The compiler lost context when I tried `.map{($0, $1)}`

Reply:
Try `.map { ($0.0, $0.1) }`

Reply:
Oh yeah, it's not an array. Thanks!